.github/actions/yarn-install/action.yaml

```
@@ -0,0 +1,21 @@
+name: yarn-install
+runs:
+  using: composite
+  steps:
+    - name: restore_cache
+      uses: actions/cache@v3.3.1
+      with:
+        key: yarn3-packages-{{ arch }}-{{ checksum "yarn.lock" }}
+        path: ".yarn/cache"
+        restore-keys: yarn3-packages-{{ arch }}-{{ checksum "yarn.lock" }}
```

If it wanted to use [GitHub Actions Context](https://docs.github.com/en/actions/learn-github-actions/contexts) feature, it should be fixed manually. I didn't test it but it should be like:

```suggestion
        key: yarn3-packages-${{ runner.arch }}-${{ hashFiles("yarn.lock") }}
        path: ".yarn/cache"
        restore-keys: yarn3-packages-${{ runner.arch }}-${{ hashFiles("yarn.lock") }}
```

- `runner.arch`: https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context
- `hashFiles(...)`: https://docs.github.com/en/actions/learn-github-actions/expressions#hashfiles

---

good point, thanks